extensions/amp-story/0.1/amp-story.css

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
```

Comment:
Do you think it's worth splitting the desktop styles into a separate amp-story-desktop.css?  We'd always pull in both, but it may be more readable that way

Reply:
It could help with clutter, but my worry would be it might become an afterthought or hard to coordinate between the two. I'm leaning toward separate though.